phoenix-core/src/main/java/org/apache/phoenix/coprocessor/generated/ServerCachingProtos.java

```diff
@@ -5759,6 +5850,80 @@ public Builder clearIndexDataColumnCount() {
         return this;
       }
 
+      // optional string parentTableType = 24;
+      private java.lang.Object parentTableType_ = "";
+      /**
+       * <code>optional string parentTableType = 24;</code>
+       */
+      public boolean hasParentTableType() {
+        return ((bitField0_ & 0x00800000) == 0x00800000);
+      }
+      /**
+       * <code>optional string parentTableType = 24;</code>
+       */
+      public java.lang.String getParentTableType() {
+        java.lang.Object ref = parentTableType_;
+        if (!(ref instanceof java.lang.String)) {
+          java.lang.String s = ((com.google.protobuf.ByteString) ref)
+              .toStringUtf8();
+          parentTableType_ = s;
+          return s;
+        } else {
+          return (java.lang.String) ref;
+        }
+      }
+      /**
+       * <code>optional string parentTableType = 24;</code>
+       */
+      public com.google.protobuf.ByteString
+          getParentTableTypeBytes() {
+        java.lang.Object ref = parentTableType_;
+        if (ref instanceof String) {
+          com.google.protobuf.ByteString b = 
+              com.google.protobuf.ByteString.copyFromUtf8(
+                  (java.lang.String) ref);
+          parentTableType_ = b;
+          return b;
+        } else {
+          return (com.google.protobuf.ByteString) ref;
+        }
+      }
+      /**
+       * <code>optional string parentTableType = 24;</code>
+       */
+      public Builder setParentTableType(
```

Comment:
Indent looks little off in this function

Reply:
This is auto generated file